test/e2e/storage/utils/utils.go

```diff
@@ -710,8 +710,10 @@ func FindVolumeGlobalMountPoints(hostExec HostExec, node *v1.Node) sets.String {
 // The namespace is still tracked and ensured that gets deleted when test terminates.
 func CreateDriverNamespace(f *framework.Framework) *v1.Namespace {
 	ginkgo.By(fmt.Sprintf("Building a driver namespace object, basename %s", f.BaseName))
-	namespace, err := f.CreateNamespace(f.BaseName, map[string]string{
```

Comment:
Since we no longer use `f.Basename` as a basename for driver namespace, can you update message logged above?

Reply:
Done